blazor/treegrid/rows/rows.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+title: Rows in Blazor Tree Grid Component | Syncfusion
+description: Checkout and learn here all about rows in Syncfusion Blazor Tree Grid component and much more details.

```

Comment:
description: Check out here and learn more details about rows in the Syncfusion Blazor Tree Grid component.